components/ui/ErrorMessage.vue

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" setup>
+defineProps({
+  message: String,
+  error: Boolean,
+});
+</script>
+
+<template>
```

Comment:
pretty straight forward for me. dont create this component. 
> use label directly

Reply:
Already remove unnecessary components